draft-fossati-tls-attestation.md

```diff
@@ -939,14 +939,10 @@ confidential computing properties.
 
 ## IoT Device Onboarding
 
-In this example, an IoT is onboarded to a cloud service provider (or to a
-network operator). In this scenario there is typically no a priori
-relationship between the device and the cloud service provider that 
-will remotely manage the device.
-
-In such scenario, the cloud service provider wants to make sure
-that the device runs the correct version of firmware, has not been 
-rooted, is not emulated or cloned.
+In this scenario, an IoT device is connected to a Device Manager, which could be a cloud service provider or a network operator.
+It is important to note that there is no pre-existing relationship between the two parties.
```

Comment:
- s/relationship/trust relationship or s/relationship/secure channel? 
- s/two parties/device and the entity?
- Since "typically" was removed in rephrasing, did you intend to strictly exclude the case when some pre-existing relationship exists.

Reply:
> * s/relationship/trust relationship or s/relationship/secure channel?

I understand the previously existing text as "the DM does not trust the device."

> * s/two parties/device and the entity?

There seems to be no ambiguity, but if you think it improves clarity I am OK to make the change.

> * Since "typically" was removed in rephrasing, did you intend to strictly exclude the case when some pre-existing relationship exists.

No, I'm just saying we don't assume it.

Reply:
> I understand the previously existing text as "the DM does not trust the device."

I prefer to write it as is then. ''There is no pre-existing trust relationship" is strictly stronger statement, implying that the device also does not trust the DM, which is not your intention, right?

> No, I'm just saying we don't assume it.

How about s/It is important to note that there is no pre-existing relationship between the two parties/We assume that the remote device management entity does not trust the device.?

Reply:
WFM, thanks!